### DISCUSSION TITLE
Consistent pings

### DISCUSSION THREAD

**Discussion by Ahptt** [itzg/docker-minecraft-server#2107] [General]
I lack much knowledge about networking when it comes to docker containers. I have the autopause feature enabled on my server, and it's consistently being awaken from random IP's. I've attempted to block these on my firewall but the server continues to get knocked. Is there something I'm missing here?

**Comment by itzg** [comment]
That's certainly annoying. My only advice is to see if the knockd logs from auto pause show anything helpful about the remote IPs.

**Reply by Ahptt** [reply]
I realized that part of the issue was that I had set an internal dmz to my ip. I removed that and it has somewhat slowed the connections. However, I still have some knocks and what appears to be bots attempting to join the server. I thought that IP-banning from the server would stop the knocks but it hasn't. I'm thinking the only way to prevent this server pinging is to drop it at the router level?

```
[09:24:20] [Server thread/WARN]: Can't keep up! Is the server overloaded? Running 53376ms or 1067 ticks behind
[09:24:21] [Server thread/INFO]: com.mojang.authlib.GameProfile@7d4ab9be[id=<null>,name=ServerOverflow,properties={},legacy=false] (/132.145.xx.xx:42374) lost connection: Disconnected
[Autopause loop] Server was knocked from 132.145.xx.xx - waiting for clients or timeout
[Autopause loop] No client connected since startup / knocked - pausing
[09:26:22] [RCON Listener #1/INFO]: Thread RCON Client /127.0.0.1 started
```

**Reply by itzg** [reply]
Yes, it has to be at router level. The server has to be running for it to apply the IP bans, which is why the pings are waking it up.

**Reply by Ahptt** [reply]
Perfect, thank you for the assistance!
